tools/cli/admin.go

```diff
@@ -774,6 +774,10 @@ func newDBCommands() []cli.Command {
 					Name:  FlagReportRate,
 					Usage: "the number of shards which get handled between each emitting of progress",
 					Value: 10,
+				},
+				cli.BoolFlag{
```

Comment:
In the future we can make this more flexible. I didn't want to handle this as part of this diff.